FloripaBus/ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using System.Linq.Expressions;
+
+namespace FloripaBus
+{
+	public abstract class ViewModelBase : INotifyPropertyChanged
+	{		
+		public event PropertyChangedEventHandler PropertyChanged;
+
+		protected void Notify(string propertyName){
+			if (this.PropertyChanged != null)
+				this.PropertyChanged (this, new PropertyChangedEventArgs (propertyName));
+		}
+
+		protected void Notify<T>(Expression<Func<T>> e)

```

Comment:
I believe you could've made use of `[CallerMemberName]` attribute here so you didn't need to identify what is the property name that's calling this method.

Please see: https://msdn.microsoft.com/en-us/library/hh534540(v=vs.140).aspx

On C# 6 you can make use of the new `nameof` operator as well, so you can get the string equivalent of that type without having to write a block of code using reflection.